cmd/config-validator/main.go

```diff
@@ -131,3 +133,26 @@ func main() {
 		log.Fatal("Pipelines validation failed.")
 	}
 }
+
+func validateTag(ppDefConf, ppRepConf *pipelineconfig.Config) error {
+	getTag := func(config *pipelineconfig.Config) string {
+		const tag = "tag"
+		for _, param := range config.Defaults.Params {
```

Comment:
The logic here should me modified according variations of params slice like [in example](https://github.com/ElementalCognition/python-images/blob/main/.tekton.yaml#L21)